### DISCUSSION TITLE
How can I know how many people follow my website from rss

### DISCUSSION THREAD

**Discussion by wanghuibin0** [timqian/chinese-independent-blogs#1181] [Q&A]
How can I know how many people follow my website from rss?

**Comment by timqian** [comment]
Currently, the follower count is from https://feeds.pub. e.g. https://feeds.pub/feed/http%3A%2F%2Ffeeds.feedburner.com%2Fruanyifeng

**Reply by wanghuibin0** [reply]
Thanks for reply. 
Does it gather statistics from only https://feeds.pub/ itself? 
For instance, if I follow ruanyifeng from inoreader, does the statistics data include me?

**Reply by timqian** [reply]
Yes~

**Comment by CyrusYip** [comment]
You can find out stats of Feedly and Inoreader on https://substats.swo.moe/ .
